packages/presentation/lib/src/di/presentation_dependencies.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:get_it/get_it.dart';
+import 'package:presentation/src/app/app_bloc.dart';
+import 'package:presentation/src/navigation/app_navigator.dart';
+
+final inject = GetIt.I;
+
+void initPresentationModule() {
+  inject.registerLazySingleton<AppNavigator>(() => AppNavigator());
```

Comment:
wow, at least someone read the getit doc and tried to use something new. thanks))